app/javascript/components/import/Import.js

```diff
@@ -74,10 +86,25 @@ class Import extends React.Component {
     this.setState({ importDuplicates: value });
   }
 
+  stopImport = async () => {
+    this.setState({ isPaused: true });
```

Comment:
this.setState is asynchronous, and if the rest of the things in this function assume isPaused is true, you will sometimes see undefined behavior. Suggest wrapping the rest of the logic in a callback:
```
this.setState({ isPaused: true }, () => {
  // post-isPaused-equals-true logic
});
```